imports/plugins/included/product-variant/containers/productsContainerAdmin.js

```diff
@@ -0,0 +1,248 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { compose } from "recompose";
+import { registerComponent, composeWithTracker } from "@reactioncommerce/reaction-components";
+import { Meteor } from "meteor/meteor";
+import { ReactiveVar } from "meteor/reactive-var";
+import { Session } from "meteor/session";
+import { Tracker } from "meteor/tracker";
+import { Reaction } from "/client/api";
+import { ITEMS_INCREMENT } from "/client/config/defaults";
+import { ReactionProduct } from "/lib/api";
+import { applyProductRevision } from "/lib/api/products";
+import { Products, Tags, Shops } from "/lib/collections";
+import { Media } from "/imports/plugins/core/files/client";
+import ProductsComponent from "../components/products";
+
+const reactiveProductIds = new ReactiveVar([], (oldVal, newVal) => JSON.stringify(oldVal.sort()) === JSON.stringify(newVal.sort()));
+
+// Isolated resubscribe to product grid images, only when the list of product IDs changes
+Tracker.autorun(() => {
+  Meteor.subscribe("ProductGridMedia", reactiveProductIds.get());
+});
+
+
+/**
+ * loadMoreProducts
+ * @summary whenever #productScrollLimitLoader becomes visible, retrieve more results
+ * this basically runs this:
+ * Session.set('productScrollLimit', Session.get('productScrollLimit') + ITEMS_INCREMENT);
+ * @return {undefined}
+ */
+function loadMoreProducts() {
+  let threshold;
+  const target = document.querySelectorAll("#productScrollLimitLoader");
+  let scrollContainer = document.querySelectorAll("#container-main");
+  if (scrollContainer.length === 0) {
+    scrollContainer = window;
+  }
+
+  if (target.length) {
+    threshold = scrollContainer[0].scrollHeight - scrollContainer[0].scrollTop === scrollContainer[0].clientHeight;
+
+    if (threshold) {
+      if (!target[0].getAttribute("visible")) {
+        target[0].setAttribute("productScrollLimit", true);
+        Session.set("productScrollLimit", Session.get("productScrollLimit") + ITEMS_INCREMENT || 24);
+      }
+    } else if (target[0].getAttribute("visible")) {
+      target[0].setAttribute("visible", false);
+    }
+  }
+}
+
+const wrapComponent = (Comp) => (
+  class ProductsContainer extends Component {
+    static propTypes = {
+      canLoadMoreProducts: PropTypes.bool,
+      productsSubscription: PropTypes.object,
+      showNotFound: PropTypes.bool
+    };
+
+    constructor(props) {
+      super(props);
+      this.state = {
+        initialLoad: true
+      };
+
+      this.ready = this.ready.bind(this);
+      this.loadMoreProducts = this.loadMoreProducts.bind(this);
+    }
+
+    ready = () => {
+      if (this.props.showNotFound === true) {
+        return false;
+      }
+
+      const isInitialLoad = this.state.initialLoad === true;
```

Comment:
Can this just be `const isInitialLoad = this.state.initialLoad;`?

Reply:
The `productsContainerAdmin.js` was moved with `git mv` from `productsContainer.js` so it's a verbatim copy. Perhaps the original author intended to be triply sure that `initialLoad` was true/false and not just undefined or a string.